plutus-tx/src/PlutusTx/IsData/Class.hs

```diff
@@ -102,6 +103,37 @@ instance UnsafeFromData Builtins.BuiltinByteString where
   {-# INLINEABLE unsafeFromBuiltinData #-}
   unsafeFromBuiltinData = BI.unsafeDataAsB
 
+instance ToData Bool where
```

Comment:
What's the difference between this instance and the one generated by `makeIsDataSchemaIndexed`?